Formula/kcptun.rb

```diff
@@ -0,0 +1,122 @@
+require "language/go"
+
+class Kcptun < Formula
+  desc "Extremely simple & fast UDP tunnel based on KCP protocol"
+  homepage "https://github.com/xtaci/kcptun"
+  url "https://github.com/xtaci/kcptun/archive/v20160808.tar.gz"
+  sha256 "23436dbd51023dd802e3af936b20007fdcdf2bee71275403a2f3f8fad528dfc0"
+  head "https://github.com/xtaci/kcptun.git"
+
+  depends_on "go" => :build
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git"
+  end
+
+  go_resource "github.com/golang/snappy" do
+    url "https://github.com/golang/snappy.git"
+  end
+
+  go_resource "github.com/hashicorp/yamux" do
+    url "https://github.com/hashicorp/yamux.git"
+  end
+
+  go_resource "github.com/urfave/cli" do
+    url "https://github.com/urfave/cli.git"
+  end
+
+  go_resource "github.com/xtaci/kcp-go" do
+    url "https://github.com/xtaci/kcp-go.git"
+  end
+
+  go_resource "github.com/klauspost/crc32" do
+    url "https://github.com/klauspost/crc32.git"
+  end
+
+  go_resource "github.com/klauspost/reedsolomon" do
+    url "https://github.com/klauspost/reedsolomon.git"
+  end
+
+  go_resource "github.com/klauspost/cpuid" do
+    url "https://github.com/klauspost/cpuid.git"
+  end
+
+  go_resource "golang.org/x/net" do
+    url "https://github.com/golang/net.git"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/xtaci"
+    ln_sf buildpath, buildpath/"src/github.com/xtaci/kcptun"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-ldflags", "-X main.VERSION=#{version} -s -w",
+      "-o", "kcptun_client", "github.com/xtaci/kcptun/client"
+    bin.install "kcptun_client"
+
+    (buildpath/"kcptun.sh").write <<-EOS.undent
+      #!/bin/sh
+
+      local_addr=":12948"
+      remote_addr="vps:29900"
+      key="it's a secret"
+      crypt="aes"
+      mode="fast2"
+      conn=1
+      mtu=1350
+      sndwnd=128
+      rcvwnd=1024
+      nocomp=""
+      datashard=10
+      parityshard=3
+      dscp=0
+
+    EOS
```

Comment:
Sure, I think that's something that should be addressed upstream rather than in the Homebrew formula.